README.md

```diff
@@ -64,6 +64,48 @@ However, processing all the datasets will take a long time and memory.
 
 *Note*: `poetry run` runs commands from within the virtual environment. You can also activate it with `poetry shell` and then simply run `etl ...`.
 
+#### Creating the pipeline of a new dataset
+
+These are the steps to create a data pipeline for a dataset called `example_dataset`, from an institution called
+`example_institution`, with version `YYYY-MM-DD` (where this date tag can typically be the current date when the dataset
+is being added to `etl`, or the date when the source data was released or updated):
+
+0. Activate the virtual environment (running `poetry shell`).
+1. **Create a new branch in the `walden` submodule**.
+2. **Create an ingest script** (e.g. `etl/vendor/walden/ingests/example_institution.py`) to download the data from its
+original source and upload it as a new data snapshot into the S3 `walden` bucket.
+This step can also be done manually (although it is preferable to do it via script, to have a record of how the data was
+obtained, and to be able to repeat the process in the future, for instance if another version of the data is released).
+Keep in mind that, if there is additional metadata, it should also be ingested into `walden` as part of the snapshot.
+If the data is in a single file for which you have a download link, this script may not be required: you can add
+this link directly in the index file (see next point).
+3. **Create an index file** `etl/vendor/walden/index/example_institution/YYYY-MM-DD/example_dataset.json` for the new
+dataset.
+You can simply copy another existing index file and adapt its content.
+This can be done manually, or, alternatively, the ingest script can also write one (or multiple) index files.
+4. **Run `make test` in `walden`** and make changes to ensure the new files in the repository have the right style and structure.
+5. **Create a pull request** to merge the new branch with the master branch in `walden`.
```

Comment:
Do we have a protocol that all pull requests should be reviewed? I think we should for new users (I really mean me!) until people get used to how it works. I think it's worth making it explicit that a review should be requested.